src/Test/L0/Listener/CommandSettingsL0.cs

```diff
@@ -534,7 +534,7 @@ public void PromptsForUserName()
                     .Setup(x => x.ReadValue(
                         Constants.Agent.CommandLine.Args.UserName, // argName
                         StringUtil.Loc("UserName"), // description
-                        true, // secret
+                        false, // not secret
                         string.Empty, // defaultValue
```

Comment:
You're comment is accurate, yet diverges from the pattern 1) within the rest of the file, and 2) even within the same method call (note `unattened` is the param name) :)

I originally tried to use named parameters for readability, but the compiler complained because of the lambda or something.